modules/features/taskerplugin/src/main/java/au/com/shiftyjelly/pocketcasts/taskerplugin/controlplayback/InputControlPlayback.kt

```diff
@@ -11,13 +11,16 @@ import com.joaomgcd.taskerpluginlibrary.input.TaskerInputRoot
 class InputControlPlayback @JvmOverloads constructor(
     @field:TaskerInputField("command") var command: String? = null,
     @field:TaskerInputField("skipToChapter") var chapterToSkipTo: String? = null,
-    @field:TaskerInputField("timeToSkipToSeconds") var timeToSkipToSeconds: String? = null
+    @field:TaskerInputField("timeToSkipToSeconds") var timeToSkipToSeconds: String? = null,
+    @field:TaskerInputField("timeToSkipSeconds") var timeToSkipSeconds: String? = null
```

Comment:
I wonder if changing these to something shorter like `skipSeconds` and `skipToSeconds` might make it harder to mix these up. The fact that the only difference between them currently is that one has a second "To" in the name makes them look very similar (to my eyes at least). I don't feel strongly though, feel free to leave it as-is if you prefer.